File: Short/bubble_short.py
def bubbleShort(n):
    for i in range(len(n)):
        for j in range(i+1,len(n)):
            if(a[i]>a[j]):
                a[i],a[j]= a[j],a[i]
    print(a)

a=[]
n=int(input("how many numbers you want"))
print("enters numbers")
for i in range(n):
    s=int(input())
    a.append(s)
bubbleShort(a)

File: Short/bubble_short.c
#include <stdio.h>

void bubbleSort(int arr[], int n) {
    for (int i = 0; i < n; i++) {
        for (int j = i + 1; j < n; j++) {
            if (arr[i] > arr[j]) {
                int temp = arr[i];
                arr[i] = arr[j];
                arr[j] = temp;
            }
        }
    }
    printf("Sorted Array: ");
    for (int i = 0; i < n; i++) {
        printf("%d ", arr[i]);
    }
}

int main() {
    int n;
    printf("How many numbers do you want? ");
    scanf("%d", &n);
    
    int arr[n];
    printf("Enter numbers:\n");
    for (int i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }
    
    bubbleSort(arr, n);
    return 0;
}

